.pre-commit-config.yaml

```
@@ -0,0 +1,18 @@
+ci:
+  autoupdate_commit_msg: "chore: [pre-commit.ci] pre-commit autoupdate"
+  autoupdate_schedule: "monthly"
+
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
```

```suggestion
    rev: v4.0.1
```

`v4.0.1` is out now. You can get `pre-commit` to update for you by doing (locally)

```console
$ pre-commit autoupdate
```

which is really nice. :)

---

Thanks for the tip, this is actually really convenient.